presto-native-execution/README.md

```diff
@@ -161,7 +161,7 @@ Run CLion:
 
 ### Run Presto Coordinator + Worker
 * Note that everything below can be done w/o using IDEs by running command line commands (not in this readme).
-* Run 'HiveQueryRunnerExternal' from IntelliJ and wait until it started (`======== SERVER STARTED ========` in the log output).
+* Run 'HiveExternalWorkerQueryRunner' from IntelliJ and wait until it started (`======== SERVER STARTED ========` in the log output).
```

Comment:
```suggestion
* Run `HiveExternalWorkerQueryRunner` from IntelliJ and wait until it starts (`======== SERVER STARTED ========` is displayed in the log output).
```
Nit formatting suggestion, and a rephrasing suggestion that has nothing to do with your change.